libs/packages/components/src/lib/click-outside/click-outside.directive.ts

```diff
@@ -1,4 +1,4 @@
-import { Directive, ElementRef, Output, EventEmitter, HostListener } from '@angular/core';
+import { Directive, ElementRef, Output, EventEmitter, HostListener, Input } from '@angular/core';
```

Comment:
Doesn't look like this import is being used

Reply:
input removed